packages/semver/src/schematics/ng-add/index.ts

```diff
@@ -0,0 +1,48 @@
+import { chain, Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+import { addPackageJsonDependency, NodeDependency, NodeDependencyType } from '@schematics/angular/utility/dependencies';
+import { of } from 'rxjs';
+import { concatMap, map } from 'rxjs/operators';
+
+import { getLatestNodeVersion } from '../../utils/npm';
+import { SchemaOptions } from './schema';
+
+const PACKAGE_NAME = '@jscutlery/semver';
+
+function addDependency(): Rule {
+  return (tree, ctx): any => {
+    return of(PACKAGE_NAME).pipe(
+      concatMap((dep) => getLatestNodeVersion(dep)),
```

Comment:
Is this necessary?
I think the schematic & builder version should be the same.
This will let users choose the version they want and they won't have any surprise as it will allow usages like `ng add @jscutlery/semver@next`

Reply:
How we grab the latest version if users don't specify it? We can hard code it but it needs maintenance on every release.

Reply:
What about the package.json?